From 13c8efb5c8b1e4dd41db08431b296bb1f4b84a06 Mon Sep 17 00:00:00 2001
From: Bruce Ashfield <bruce.ashfield@windriver.com>
Date: Fri, 22 Mar 2013 20:48:33 -0700
Subject: [PATCH 06/10] ck: make 1000HZ the default

Imported from: http://ck.kolivas.org/patches/3.0/3.8/3.8-ck1/patches/hz-default_1000.patch

Set default HZ to 1000 which is what most desktop users should still be using.

-ck

Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 kernel/Kconfig.hz |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/Kconfig.hz b/kernel/Kconfig.hz
index 94fabd5..b0c107e 100644
--- a/kernel/Kconfig.hz
+++ b/kernel/Kconfig.hz
@@ -4,7 +4,7 @@
 
 choice
 	prompt "Timer frequency"
-	default HZ_250
+	default HZ_1000
 	help
 	 Allows the configuration of the timer frequency. It is customary
 	 to have the timer interrupt run at 1000 Hz but 100 Hz may be more
-- 
1.7.10.4

